libs/langchain-anthropic/src/chat_models.ts

```diff
@@ -739,7 +753,10 @@ export class ChatAnthropicMessages<
           if (data?.usage !== undefined) {
             usageData.output_tokens += data.usage.output_tokens;
           }
-        } else if (data.type === "content_block_delta") {
+        } else if (
+          data.type === "content_block_delta" &&
+          data.delta.type === "text_delta"
```

Comment:
Why necessary now? We don't support streaming tool calls yet.

Reply:
SDK types bump req it to access text below